packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/MultiFrameViewerContainer.js

```diff
@@ -76,9 +76,9 @@ function MultiFrameViewerContainer({
 }) {
   const [dragMove, setDragMove] = useState()
   // TODO: replace this with a better function to parse the image location from a subject.
-  const imageUrl = subject ? Object.values(subject.locations[frame])[0] : null
+  const imageLocation = subject ? subject.locations.find(l => l.type === 'image') : null
```

Comment:
Clicking a different frame doesn't change the image as expected. Looks like missing reference to frame per subject viewer store in `imageLocation` declaration.

```suggestion
  const imageLocation = subject?.locations[frame].type === 'image' ? subject.locations[frame] : null
```

Reply:
```suggestion
  const imageLocation = subject ? subject.locations[frame] : null
```

Reply:
Good catch.